examples/abitest/proto/abitest.proto

```diff
@@ -32,10 +32,29 @@ message ABITestResponse {
     string name = 1;
     bool success = 2;
     string details = 3;
+    bool disabled = 4;
   }
   repeated TestResult results = 1;
 }
 
+message GrpcTestRequest {
+  oneof method_result {
+    int32 err_code = 1;
+    string text = 2;
+  }
+}
+
+message GrpcTestResponse {
+  string text = 1;
+}
+
 service OakABITestService {
   rpc RunTests(ABITestRequest) returns (ABITestResponse);
+
+  // The following methods allow testing of the gRPC connection to an Oak
+  // Application.
+  rpc UnaryMethod(GrpcTestRequest) returns (GrpcTestResponse);
```

Comment:
Can you clarify what these methods are supposed to do? i.e. reflect back the error to the caller.